source/change-streams/tests/unified/change-streams-errors.yml

```diff
@@ -0,0 +1,159 @@
+description: "change-streams errors"
+schemaVersion: "1.0"
+createEntities:
+  - client:
```

Comment:
I'm finding the numerous client, database and collection entities here confusing. Don't you just need one of each for the following tests?

Reply:
This was a static preamble in the converter script; turns out only `change-streams.yml` needs all of these, so I've pruned it in the other files.